src/components/swimlanes/swimLanesLoading.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  SkeletonLoader,
+  SimpleGrid,
+} from "@nypl/design-system-react-components";
+
+const SwimLanesLoading = () => {
+  return (
+    <SimpleGrid columns={4}>
+      <SkeletonLoader imageAspectRatio="landscape" isBordered contentSize={1} />
```

Comment:
Add a "title" skeleton loader with `showImage={false}`, `contentSize={0}`, and `headingSize={1}`.

Reply:
Adding a title skeleton loader title will throw off the `SimpleGrid`'s four columns. So the DOM structure will have to be reworked a bit.